src/qmake/platforms/windows.pri

```diff
@@ -89,3 +89,66 @@ HEADERS += \
     platforms/windows/windowsstartatbootwatcher.h \
     systemtraynotificationhandler.h \
     wgquickprocess.h
+
+
+# Installs for the /unsigned folder
+
+mvpnExe.files = $$PWD/../../../mozillavpn.exe
+mvpnExe.path = $$PWD/../../../unsigned/
```

Comment:
I guess this doesn't work. you need to add:
`mvpnExe.CONFIG = no_check_exist executable`